From 2a4c0c470955d78657003cb613a854e77cd92133 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 06 Sep 2018 09:46:34 +0300
Subject: [PATCH] Makefile: add btl to the release target

btl is one of the embedded repositories and should be treated as
others in terms of release.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 3ee46d8..6659957 100644
--- a/Makefile
+++ b/Makefile
@@ -66,7 +66,7 @@
 endif
 
 #TODO: add btl once it is separated from br-ext
-RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader
+RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader btl
 RELEASE_TAG_POINTER := latest-embedded-release
 
 .PHONY: gaudi-release goya-release
